test/shared_test/test_ortops_azure.cc

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifdef ENABLE_AZURE
+#include <cstdlib>
+
+#include "gtest/gtest.h"
+
+#include "ocos.h"
+#include "narrow.h"
+#include "test_kernel.hpp"
+#include "utils.hpp"
+
+using namespace ort_extensions;
+using namespace ort_extensions::test;
+
+// Test custom op with OpenAIAudioInvoker calling Whisper
+// Default input format. No prompt.
+TEST(AzureOps, OpenAIWhisper_basic) {
+  const char* auth_token = std::getenv("OPENAI_AUTH_TOKEN");
```

Comment:
Any plan to enable it in extensions.ci, or not?

Reply:
yes but need to figure out how to test these things properly. easiest would be a stub https server that returns canned results so there's no need for a real auth token.